.circleci/config.yml

```diff
@@ -6,19 +6,19 @@ orbs:
 parameters:
   image_suffix:
     type: string
-    default: '-v186a1be'
+    default: '-dev-51cc45e'
```

Comment:
I will update this Docker image tag suffix after https://github.com/citusdata/the-process/pull/97 lands